label_studio/io_storages/utils.py

```diff
@@ -109,3 +116,73 @@ def parse_range(range_header):
         end = ''
 
     return start, end
+
+
+@dataclass
+class StorageObjectParams:
```

Comment:
I think `StorageObject` would be a better name for this, since it actually contains the task now